example/src/main/java/com/thefuntasty/mvvmsample/domain/GetUserFullNameObservabler.kt

```diff
@@ -1,16 +1,15 @@
 package com.thefuntasty.mvvmsample.domain
 
-import com.thefuntasty.interactors.BaseObservabler
+import com.thefuntasty.interactors.interactors.BaseObservabler
 import com.thefuntasty.mvvmsample.data.store.UserStore
 import io.reactivex.Observable
 import javax.inject.Inject
 
 class GetUserFullNameObservabler @Inject constructor(
     private val userStore: UserStore
-) : BaseObservabler<String>() {
+) : BaseObservabler<Unit, String>() {
 
-    override fun prepare(): Observable<String> {
-        return userStore.getUser()
-            .map { "${it.firstName} ${it.lastName}" }
+    override fun prepare(args: Unit): Observable<String> {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/abd629572327cbabcfd5483dec148fd3bc6b5ac2/example/src/main/java/com/thefuntasty/mvvmsample/domain/GetUserFullNameObservabler.kt#L12"></span>Functions with exact one statement, the return statement, can be rewritten with ExpressionBodySyntax.</td>
    </tr>
  </tbody>
</table>